Jenkinsfile

```diff
@@ -12,6 +12,9 @@ node {
         sh("yarn")
         sh("yarn run lint")
       }
+      stage("Lint Factorybot") {
+        govuk.runTests("factorybot_lint")
```

Comment:
Since we're not actually running tests here we should probably just use the sh function. Otherwise that's going to conflict with the statsd timing in: https://github.com/alphagov/govuk-jenkinslib/blob/d2bb2e55c779dacdead1b290031c2c3b07bffa1f/vars/govuk.groovy#L652